addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -303,6 +303,21 @@ MockServer.include({
         }
         return this._super(route, args);
     },
+    /**
+     * @override
+     */
+    _mockCreate(model, values) {
+        let values2;
+        if (model === 'mail.channel') {
+            values2 = {
+                ...values,
+                channel_partner_ids: values.channel_partner_ids || [this.currentPartnerId],
+            };
+        } else {
+            values2 = values;
+        }
+        return this._super(model, values2);
+    },
```

Comment:
try to define the default on the field, that should work the same without this extra code